### DISCUSSION TITLE
Cannot see ai completion suggestions after update

### DISCUSSION THREAD

**Discussion by higgsboz** [LazyVim/LazyVim#5199] [Q&A]
Just updated my nvim config which introduced blink and supermaven extras. I removed my nvim-cmp config, added the blink and supermaven extras, but can't see the suggestions any longer.

The completion is still working fine, I just can't see it before I press tab.

Example video:

https://github.com/user-attachments/assets/2c6fd208-c15b-4653-b117-a91987411241

**Comment by dpetka2001** [comment]
Fresh LazyVim installation with supermaven Extra enabled. Screencast following

[Screencast_2024-12-19-19-06-41.webm](https://github.com/user-attachments/assets/69215301-0913-4085-84a3-ab8b0dfb8771)

Seems to be working as expected. 

What you seem to be experiencing in your video is the supermaven ghost-text, which is disabled by default because it's showing the completion in the blink auto-completion menu by default. If you want to see the ghost text and not the completion in blink's auto-completion menu, then set `vim.g.ai_cmp = false` in your `options.lua` file.

**Reply by higgsboz** [reply]
Ah perfect, I appreciate the response! I didn't realize the ai suggestions were still there if I started to type. But knowing about that option is helpful. 

Thank you!

**Comment by higgsboz** [comment]
I wanted to reopen this discussion, because even though this suggestion fixed the ghost text not showing, I feel like we should be able to have both.

- Able to tab to complete ghost text
- Able to select an entry from blink-cmp to complete as well

Right now, it's limited to one or the other
